v2.0/releases/rn.md

```diff
@@ -6,12 +6,50 @@ category: release
 # TiDB 版本发布历史
 
 TiDB 历史版本发布声明如下：
+## 3.0
```

Comment:
Please add a blank line before this line.

Reply:
Done~